### PR TITLE
Set up a "Hello world" project with two apps

### PR DESCRIPTION
**Pull request by cvalencia1991**
# In this milestone, I did the following activities 
- Create a new rails API 
- Create controller, model for the greetings 
- Create randomly Greetings in the seed 
- Create API

### CODE REVIEW

File: config/initializers/cors.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# Read more: https://github.com/cyu/rack-cors
+
+# Rails.application.config.middleware.insert_before 0, Rack::Cors do
+#   allow do
+#     origins "example.com"
+#
+#     resource "*",
+#       headers: :any,
+#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
+#   end
```
Comment (Munsa1) [inline config/initializers/cors.rb]: - [ ] Kindly setup and confiure your `cors` as it is currently preventing the random message from bein displayed. Checkout this [resource](https://www.stackhawk.com/blog/rails-cors-guide/)

**Review by Munsa1 — CHANGES_REQUESTED** [review]
Hi @cvalencia1991, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights
Passing Linters :heavy_check_mark: 
Correct GitHub Flow :heavy_check_mark: 
Descriptive ReadMe File :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/initializers/cors.rb (hunk 1 above)
Comment (cvalencia1991) [reply config/initializers/cors.rb]: thank you, you must the first CR that send me an information to unstuck, thanks

**Review by DeliceLydia — APPROVED** [review]
## Hi @cvalencia1991, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
# Congratulations! 🎉 

## Highlights
- Greetings are being fetched well

### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by cvalencia1991** [comment]
Thank you :smile:

**Pull request merged**